src/static/js/utils/date-translate.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// If 0 = January 2000, then we will translate an int to a month and year!
```

Comment:
Yeah, I saw `formatDates.js`, but I wasn't sure (in the 15 seconds I read over it) if it was doing what I needed. I had already figured out how to do the date format I was using (which was a string akin to `2009-01-01`). So I just went with what I had and figured we could reconcile it later. :D